.github/workflows/on-demand.yml

```diff
@@ -0,0 +1,33 @@
+name: OnDemand Lunatic CI (publish every branch)
```

Comment:
does the ci publish every branches or just the selected branch when running the workflow?